analysis/question_1/top5_countries_suicide_rate.py

```diff
@@ -0,0 +1,43 @@
+# This code is the analysis on the top 5 countries with hight suicide rates
+# It further attempts to explore their differences and similarity of suicide rates in different age groups
+# barplot from Seaborn library is used for visualisation 
+
+import os
+import pandas as pd
+import matplotlib.pyplot as plt
+import numpy as np
+import seaborn as sns
+
+# Getting Data of crude suicide rate
+suicide_rate_data_filepath = os.path.join("..\\processed_data", "crude_suicide_rates.csv")
```

Comment:
better use os.path.join("..", "processed_data", "crude_suicide_rates.csv") here

Reply:
Thanks, Ming. This really helps.